sass/compat/plugins/woo-carrier-agents/_checkout-shipping-methods.scss

```diff
@@ -0,0 +1,16 @@
+// ------------------------------------
+// CHECKOUT SHIPPING METHODS
+// ------------------------------------
+
+body.woocommerce-checkout .fc-content .carrier-agent {
```

Comment:
- Selectors are specified for the checkout page, but are these shipping methods from this plugin also visible on the cart page?

If so, these styles would have to be duplicated for the cart page. In that case, it would be better to set a local SASS variable at the level of the `_import` files and use them here. Similarly to how we do it for the theme compat styles.

Reply:
Those aren't visible on the cart page both with FC PRO cart feature enabled and without.